app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,20 @@
+class ReviewsController < ApplicationController
+  load_and_authorize_resource
+
+  def index; end
+
+  def create
+    @review = current_user.reviews.new(review_params)
```

Comment:
Please add cancancan to check if user have a permission to create a review
https://github.com/CanCanCommunity/cancancan/wiki/Authorizing-controller-actions